docs/griptape-framework/structures/src/config_defaults.py

```diff
@@ -0,0 +1,12 @@
+from griptape.configs import Defaults
+from griptape.configs.drivers import AnthropicDriversConfig, OpenAiDriversConfig
+from griptape.drivers.prompt.anthropic_prompt_driver import AnthropicPromptDriver
+from griptape.structures import Agent
+
+Defaults.drivers_config = OpenAiDriversConfig()  # Default
+openai_agent = Agent()
+
+Defaults.drivers_config = AnthropicDriversConfig()
```

Comment:
Not sure if it is currently possible, but I'd love to see an example of "clearing" the defaults, so that an error is thrown if you don't set a prompt driver for a specific use case. In some instances, I want to be sure that I'm not accidentally relying on a driver that attempts to makes a network call that I didn't want because of some default that I missed or didn't think about.

Maybe this exists via a `DummyDriversConfig`?

Reply:
That's what [DriversConfig](https://github.com/griptape-ai/griptape/blob/9d9b643976f30fdf0af16b75c9b92503b076888e/griptape/configs/drivers/drivers_config.py#L34-L65) provides -- it's all Dummy Drivers. The ["Custom"](https://docs.griptape.ai/stable/griptape-framework/structures/configs/#custom) portion of this doc touches on this.